features/core/src/test/java/org/apache/karaf/features/internal/download/impl/DownloadManagerHelperTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.karaf.features.internal.download.impl;
```

Comment:
This file doesn't contain the Apache license header, can you please add it ?

Reply:
sure, done